### PR TITLE
feat(NcRichContenteditable): Allow to pass in an icon URL

### PR DESCRIPTION
**Pull request by nickvergessen**
- Required for https://github.com/nextcloud/spreed/issues/9188
- Talk PR to show it in action https://github.com/nextcloud/spreed/pull/9332

### CODE REVIEW

File: src/components/NcRichContenteditable/NcAutoCompleteResult.vue [hunk 1]
```diff
@@ -67,6 +67,10 @@ export default {
 			type: String,
 			required: true,
 		},
+		iconUrl: {
```
Comment (raimund-schluessler) [inline src/components/NcRichContenteditable/NcAutoCompleteResult.vue:70]: Since this is the avatar url, I would also call it `avatarUrl`.

Comment (nickvergessen) [reply src/components/NcRichContenteditable/NcAutoCompleteResult.vue:70]: Then it conflicts with the computed and the CSS avatar option is also called `icon` so I tried to keep it simple and inline

Comment (Antreesy) [reply src/components/NcRichContenteditable/NcAutoCompleteResult.vue:70]: We could come up with something like `predefinedUrl`, as we provide here a ready-to-download URL

**Review by Antreesy — APPROVED** [review]
Works properly, shouldn't break anything

**Pull request merged**